### PR TITLE
Update tbots.py script with new arguments

### PR DESCRIPTION
**Pull request by nimazareian**
### Description
Add optimized build argument `-o` (-> `--copt=-O3`) to the `./tbots.py` script.

Add the ability to flash multiple robots easier. Flashing all 8 robots:

Before: 

```bash
bazel run //software/jetson_nano/ansible:run_ansible --cpu=jetson_nano --copt=-O3 -- -pb deploy_nano.yml --hosts  192.168.0.200 192.168.0.201 192.168.0.202 192.168.0.203 192.168.0.204 192.168.0.205 192.168.0.206 192.168.0.207 -pwd=<nano_pwd>
```

Now:

```bash
./tbots.py run run_ansible -flash_robots 0 1 2 3 4 5 6 7 -pwd <nano_pwd>
```

This change will also automatically compile Thunderloop in optimized mode when the `-f` argument is passed.

I'm open to suggestions and other ideas!

### Testing Done
Files still build and run with bazel and tbots.py script.

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: docs/useful-robot-commands.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ This will stop the current Systemd services, replace and restart them. Binaries
...
+
+`./tbots.py run run_ansible -f <robot_ids> -pwd <jetson_nano_password>` (Note that this uses robot IDs rather than full robot IP addresses)
+
+Example: Flashing robots 1, 4, and 7
+
+`./tbots.py run run_ansible -f 1 4 7 -pwd <jetson_nano_password>`
+
 ## Flashing the powerboard
 
 This will flash powerloop, the current firmware in `software/power/`, onto the power board. It will prompt the user into setting the powerboard into bootloader mode by holding the reset button and pressing the boot button. Then once the board is flashed, pressing the reset button after to use the new firmware.
```
Comment (PhilD71) [inline docs/useful-robot-commands.md]: Is this going to flash powerloop every time? 

Also I think you have the buttons backwards according to this message from Dannon in #robot_integration

**Review by PhilD71 — COMMENTED** [review]
Nice! One small nit, but otherwise looking good. Happy to see the --cpu=jetson_nano make its way in. Won't waste another evening of testing just because I forgot that flag!

File: docs/useful-robot-commands.md (hunk 1 above)
Comment (nimazareian) [reply docs/useful-robot-commands.md]: So flashing powerloop is done separately and not done through the `deploy_nano` script. 
But regarding the button presses, I'm personally not sure. @LiCody Can you confirm if the docs is correct?

Comment (PhilD71) [reply docs/useful-robot-commands.md]: Ah whoops rookie mistake - that's not even part of the PR.

**Review by PhilD71 — DISMISSED** [review]

File: docs/useful-robot-commands.md (hunk 1 above)
Comment (LiCody) [reply docs/useful-robot-commands.md]: oh the docs are wrong should be hold boot press reset

**Review by LiCody — DISMISSED** [review]

File: docs/useful-robot-commands.md (hunk 1 above)
Comment (nimazareian) [reply docs/useful-robot-commands.md]: I've updated the docs

Comment (PhilD71) [reply docs/useful-robot-commands.md]: Thanks for updating even tho it wasn't ur change!

**Review by PhilD71 — APPROVED** [review]
looks good!

**Review by LiCody — APPROVED** [review]

**Review by JoshuaWu7 — APPROVED** [review]
LGTM

**Review by itsarune — APPROVED** [review]
nice work!

**Pull request merged**